src/libcore/fmt/mod.rs

```diff
@@ -455,22 +455,18 @@ impl Display for Arguments<'_> {
 ///
 /// impl fmt::Debug for Point {
 ///     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-///         write!(f, "Point {{ x: {}, y: {} }}", self.x, self.y)
+///         f.debug_struct("Point")
+///          .field("x", &self.x)
+///          .field("y", &self.y)
+///          .finish()
 ///     }
 /// }
 ///
 /// let origin = Point { x: 0, y: 0 };
 ///
-/// println!("The origin is: {:?}", origin);
-/// ```
```

Comment:
This line shouldn't have been deleted. The code block is continuing into the prose.

Reply:
Thank you! 
I guess I need to make my IDE highlight these stuff.